src/App.vue

```diff
@@ -978,6 +1006,31 @@ export default class App extends Mixins(
     await updateLdUser(key, email, firstName, lastName, custom)
   }
 
+  /** Stores business info from Legal API. */
+  async storeBusinessInfo (response: any): Promise<void> {
+    const business = response?.data?.business as BusinessIdIF
+
+    // if (!business) {
+    //   throw new Error('Invalid business info')
+    // }
```

Comment:
This is the only check that the API returned `response.data.business`. I think you should uncomment this.